orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -564,13 +599,57 @@ def _enable_conditional_optimizations(
                 # Enable sparsity-based optimization when applicable.
                 if len(label_sparsity_results) > 0:
                     graph_transformer_config.sparse_label_input_names = label_sparsity_results
-                    logger.info(f"Label sparsity based optimization is on for {label_sparsity_results}")
+                    self._logger.info("Label sparsity based optimization is on for %s", label_sparsity_results)
+                    self._feature_map.append(
+                        [" - Label Sparsity", sparse_opt_status, "Label inputs: " + str(label_sparsity_results)]
+                    )
 
                 if len(embed_sparsity_results) > 0:
                     graph_transformer_config.sparse_embedding_input_names = embed_sparsity_results
-                    logger.info(f"Embedding sparsity based optimization is on for {embed_sparsity_results}")
+                    self._logger.info("Embedding sparsity based optimization is on for %s", embed_sparsity_results)
+                    self._feature_map.append(
+                        [
+                            " - Embedding Sparsity",
+                            sparse_opt_status,
+                            "Embedding inputs: " + str(embed_sparsity_results),
+                        ]
+                    )
 
             # If users don't want to print input density, disable the input density observer to avoid overhead
             # when looping through inputs during training.
             if not self._print_input_density:
                 self._rt_inspector.disable_input_inspector()
+
+    def _log_feature_stats(self):
+        rank = 0
+        if torch.distributed.is_initialized():
+            rank = torch.distributed.get_rank()
+
+        if rank != 0:
+            return
+
+        self._feature_map.append(
+            [
+                "Auto Fallback",
+                "ON" if self._fallback_manager.policy is not _FallbackPolicy.FALLBACK_DISABLE else "OFF",
+                "Fallback to PyTorch when encountering unsupported ops",
+            ]
+        )
+
+        mode = "training" if self._export_mode == torch.onnx.TrainingMode.TRAINING else "inference"
+        mode = f"{_logger.LogColor.UNDERLINE}{mode}{_logger.LogColor.ENDC}"
+
+        stat = f"\n\n{_logger.LogColor.HEADER}***** ONNX Runtime (ORTModule) is accelerating your model *****{_logger.LogColor.ENDC}\n\n"
```

Comment:
nit: ONNX Runtime Training (ORTModule) accelerates training your model

Reply:
ORTModule not only accelerate training, but also evaluating, so I avoided using "training your model" in the title here. I changed to `ONNX Runtime Training (ORTModule) is accelerating your model`